### DISCUSSION TITLE
[Feature]: Example for how to correctly combine `<TabsContent>` with `<NuxtPage>`

### DISCUSSION THREAD

**Discussion by DamianGlowala** [unovue/reka-ui#1337] [Example Request]
### Describe the feature

I am using Nuxt and have a use case with a tabs component, where each tab should correspond to a different URL/page. Could we please have an example of how to correctly integrate these two (or maybe there's someone who achieved it already and would be kind to share)?

### Additional information

- [ ] I intend to submit a PR for this feature.
- [ ] I have already implemented and/or tested this feature.

**Comment by zernonia** [comment]
Interesting request. We can have a Stackblitz demo create showing this interaction between pages and tabs component!

Can we move this to [Example request](https://github.com/radix-vue/radix-vue/discussions/categories/example-request) discussion too?

**Comment by DamianGlowala** [comment]
Sure, let's move this one too! 😄 I made several attempts to achieve this already, but with some pitfalls. Really curious how your example would look like!
